python/src/trezorlib/ethereum.py

```diff
@@ -225,29 +253,41 @@ def sign_tx_eip1559(
     )
 
     response = client.call(msg)
+    assert isinstance(response, messages.EthereumTxRequest)
 
     while response.data_length is not None:
         data_length = response.data_length
         data, chunk = data[data_length:], data[:data_length]
         response = client.call(messages.EthereumTxAck(data_chunk=chunk))
+        assert isinstance(response, messages.EthereumTxRequest)
 
+    assert response.signature_v is not None
+    assert response.signature_r is not None
+    assert response.signature_s is not None
     return response.signature_v, response.signature_r, response.signature_s
 
 
 @expect(messages.EthereumMessageSignature)
-def sign_message(client, n, message):
-    message = normalize_nfc(message)
-    return client.call(messages.EthereumSignMessage(address_n=n, message=message))
+def sign_message(
+    client: "TrezorClient", n: "Address", message: AnyStr
+) -> "MessageType":
+    return client.call(
+        messages.EthereumSignMessage(address_n=n, message=normalize_nfc(message))
+    )
 
 
 @expect(messages.EthereumTypedDataSignature)
 def sign_typed_data(
-    client, n: List[int], data: Dict[str, Any], *, metamask_v4_compat: bool = True
-):
+    client: "TrezorClient",
+    n: "Address",
+    data: Dict[str, Any],
+    *,
+    metamask_v4_compat: bool = True,
+) -> "MessageType":
     data = sanitize_typed_data(data)
     types = data["types"]
 
-    request = messages.EthereumSignTypedData(
+    request: Any = messages.EthereumSignTypedData(
```

Comment:
`MessageType` instead of `Any`, at least

Reply:
Deleted the annotation completely, it was not necessary - in b70acaf